.github/workflows/tag-publish.yml

```diff
@@ -1,122 +1,81 @@
           SEMVER_PART=${{ github.event.inputs.bump }}
+          CHECKOUT_BRANCH=${{ github.event.inputs.branch }}
         fi
         echo ::set-output name=semver-part::$SEMVER_PART
+        echo ::set-output name=checkout-branch::$CHECKOUT_BRANCH
     - name: Checkout current code
       uses: actions/checkout@v2
       with:
-        ref: dev
-        token: ${{ secrets.BROADBOT_GITHUB_TOKEN }}
-    - name: Pull Vault image
-      run: docker pull vault:1.1.0
-    # Currently, there's no way to add capabilities to Docker actions on Git, and Vault needs IPC_LOCK to run.
-    - name: Get Vault token
-      id: vault-token-step
-      run: |
-        VAULT_TOKEN=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault write -field token \
-            auth/approle/login role_id=${{ secrets.VAULT_APPROLE_ROLE_ID }} \
-            secret_id=${{ secrets.VAULT_APPROLE_SECRET_ID }})
-        echo ::set-output name=vault-token::$VAULT_TOKEN
-        echo ::add-mask::$VAULT_TOKEN
-    - name: Get Vault secrets
-      id: vault-secret-step
-      run: |
-        ARTIFACTORY_USERNAME=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault read -field username ${ARTIFACTORY_ACCOUNT_PATH})
-        ARTIFACTORY_PASSWORD=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault read -field password ${ARTIFACTORY_ACCOUNT_PATH})
-        GCR_EMAIL=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault read -field ci-gcr-sa-email ${VAULT_PATH_GCR})
-        GCR_KEY=$(docker run --rm --cap-add IPC_LOCK \
-          -e "VAULT_TOKEN=${{ steps.vault-token-step.outputs.vault-token }}" \
-          -e "VAULT_ADDR=${VAULT_ADDR}" \
-          vault:1.1.0 \
-          vault read -field ci-gcr-sa-key ${VAULT_PATH_GCR})
-        echo ::set-output name=gcr-email::$GCR_EMAIL
-        echo ::add-mask::$GCR_EMAIL
-        echo ::set-output name=gcr-key::$GCR_KEY
-        echo ::add-mask::$GCR_KEY
-        echo ::set-output name=artifactory-username::$ARTIFACTORY_USERNAME
-        echo ::add-mask::$ARTIFACTORY_USERNAME
-        echo ::set-output name=artifactory-password::$ARTIFACTORY_PASSWORD
-        echo ::add-mask::$ARTIFACTORY_PASSWORD
-    - name: "Bump the tag to a new version"
-      uses: databiosphere/github-actions/actions/bumper@v0.0.3
+        ref: ${{ steps.controls.outputs.checkout-branch }}
+        token: ${{ secrets.REPO_ACCESS_TOKEN }}
+    - name: Bump the tag to a new version
+      uses: databiosphere/github-actions/actions/bumper@-0.0.3
```

Comment:
I believe this needs to be updated to `databiosphere/github-actions/actions/bumper@bumper-0.0.3` to pick up your recent changes. The tag also includes the action name since there are multiple actions in that repo.